frontend/src/scenes/persons/cohortLogic.js

```diff
@@ -46,14 +38,22 @@ export const cohortLogic = kea({
                 setToastId: (_, { toastId }) => toastId,
             },
         ],
+        lastSavedAt: [
+            false,
+            {
+                setLastSavedAt: (_, { lastSavedAt }) => lastSavedAt,
+            },
+        ],
     }),
 
     listeners: ({ sharedListeners }) => ({
         saveCohort: async ({ cohort }) => {
-            if (cohort.id) {
+            if (cohort.id !== 'new') {
                 cohort = await api.update('api/cohort/' + cohort.id, cohort)
+                cohortsModel.actions.updateCohort(cohort)
```

Comment:
@mariusandra Need some kea magic. When calling this it gives me the following error:

![image](https://user-images.githubusercontent.com/1727427/100662388-215e5880-3355-11eb-970c-8bf790440c96.png)

From [Kea's docs](https://kea.js.org/docs/guide/additional#connecting-logic-together) i believe this should work

Reply:
Hard for it to do anything if `updateCohort` is not an action... 🤷‍♂️